GraphQLErrorOptions not supported by graphql < 16.5 in 2.1.0-alpha.1 or higher

At least with some versions of `graphql@16`, there's a build error

```
$ tsc
node_modules/@apollo/federation-internals/dist/error.d.ts:1:33 - error TS2305: Module '"graphql"' has no exported member 'GraphQLErrorOptions'.

1 import { ASTNode, GraphQLError, GraphQLErrorOptions, GraphQLFormattedError } from "graphql";
                                  ~~~~~~~~~~~~~~~~~~~

Found 1 error in node_modules/@apollo/federation-internals/dist/error.d.ts:1
```

This was introduced with an error cleanup https://github.com/apollographql/federation/pull/1914. `GraphQLErrorOptions` was introduced in `graphql@16.5.0` https://github.com/graphql/graphql-js/releases/tag/v16.5.0

This was happening with `graphql@16.3.0`. Updating to `graphql@16.5.0` fixes it.

--- a/composition-js/CHANGELOG.md
+++ b/composition-js/CHANGELOG.md
@@ -2,7 +2,7 @@
 
 This CHANGELOG pertains only to Apollo Federation packages in the 2.x range. The Federation v0.x equivalent for this package can be found [here](https://github.com/apollographql/federation/blob/version-0.x/federation-js/CHANGELOG.md) on the `version-0.x` branch of this repo.
 
-- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions`
+- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions` [PR #2060](https://github.com/apollographql/federation/pull/2060)
 
 ## 2.1.0-alpha.3
 

--- a/composition-js/CHANGELOG.md
+++ b/composition-js/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 This CHANGELOG pertains only to Apollo Federation packages in the 2.x range. The Federation v0.x equivalent for this package can be found [here](https://github.com/apollographql/federation/blob/version-0.x/federation-js/CHANGELOG.md) on the `version-0.x` branch of this repo.
 
+- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions`
+
 ## 2.1.0-alpha.3
 
 - Don't require `@link` when using `@composeDirective` [PR #2046](https://github.com/apollographql/federation/pull/2046)

--- a/composition-js/package.json
+++ b/composition-js/package.json
@@ -31,6 +31,6 @@
     "@apollo/query-graphs": "file:../query-graphs-js"
   },
   "peerDependencies": {
-    "graphql": "^16.0.0"
+    "graphql": "^16.5.0"
   }
 }

--- a/gateway-js/CHANGELOG.md
+++ b/gateway-js/CHANGELOG.md
@@ -2,7 +2,7 @@
 
 This CHANGELOG pertains only to Apollo Federation packages in the 2.x range. The Federation v0.x equivalent for this package can be found [here](https://github.com/apollographql/federation/blob/version-0.x/gateway-js/CHANGELOG.md) on the `version-0.x` branch of this repo.
 
-- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions`
+- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions` [PR #2060](https://github.com/apollographql/federation/pull/2060)
 - Upgrade underlying `@apollo/utils.fetcher` to support aborting a request. This is a type-only change, and will not impact the underlying runtime. [PR #2017](https://github.com/apollographql/federation/pull/2017).
 
 ## 2.1.0-alpha.3

--- a/gateway-js/CHANGELOG.md
+++ b/gateway-js/CHANGELOG.md
@@ -2,6 +2,9 @@
 
 This CHANGELOG pertains only to Apollo Federation packages in the 2.x range. The Federation v0.x equivalent for this package can be found [here](https://github.com/apollographql/federation/blob/version-0.x/gateway-js/CHANGELOG.md) on the `version-0.x` branch of this repo.
 
+- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions`
+- Upgrade underlying `@apollo/utils.fetcher` to support aborting a request. This is a type-only change, and will not impact the underlying runtime. [PR #2017](https://github.com/apollographql/federation/pull/2017).
+
 ## 2.1.0-alpha.3
 
 - Some TypeScript types, such as the arguments and return value of `GraphQLDataSource.process`, are defined using types from the `@apollo/server-gateway-interface` package instead of from `apollo-server-types` and `apollo-server-core`. This is intended to be fully backwards-compatible; please file an issue if this leads to TypeScript compilation issues. [PR #2044](https://github.com/apollographql/federation/pull/2044)

--- a/gateway-js/package.json
+++ b/gateway-js/package.json
@@ -45,6 +45,6 @@
     "node-fetch": "^2.6.7"
   },
   "peerDependencies": {
-    "graphql": "^16.0.0"
+    "graphql": "^16.5.0"
   }
 }

--- a/internals-js/CHANGELOG.md
+++ b/internals-js/CHANGELOG.md
@@ -1,5 +1,7 @@
 # CHANGELOG for `@apollo/federation-internals`
 
+- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions`
+
 ## 2.1.0-alpha.3
 
 - Don't require `@link` when using `@composeDirective` [PR #2046](https://github.com/apollographql/federation/pull/2046)

--- a/internals-js/CHANGELOG.md
+++ b/internals-js/CHANGELOG.md
@@ -1,6 +1,6 @@
 # CHANGELOG for `@apollo/federation-internals`
 
-- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions`
+- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions` [PR #2060](https://github.com/apollographql/federation/pull/2060)
 
 ## 2.1.0-alpha.3
 

--- a/internals-js/package.json
+++ b/internals-js/package.json
@@ -31,6 +31,6 @@
     "access": "public"
   },
   "peerDependencies": {
-    "graphql": "^16.0.0"
+    "graphql": "^16.5.0"
   }
 }

--- a/query-graphs-js/CHANGELOG.md
+++ b/query-graphs-js/CHANGELOG.md
@@ -1,6 +1,6 @@
 # CHANGELOG for `@apollo/query-graphs`
 
-- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions`
+- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions` [PR #2060](https://github.com/apollographql/federation/pull/2060)
 
 ## 2.1.0-alpha.3
 

--- a/query-graphs-js/CHANGELOG.md
+++ b/query-graphs-js/CHANGELOG.md
@@ -1,5 +1,7 @@
 # CHANGELOG for `@apollo/query-graphs`
 
+- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions`
+
 ## 2.1.0-alpha.3
 
 - Add `@defer` support [PR #1958](https://github.com/apollographql/federation/pull/1958)

--- a/query-graphs-js/package.json
+++ b/query-graphs-js/package.json
@@ -31,6 +31,6 @@
     "access": "public"
   },
   "peerDependencies": {
-    "graphql": "^16.0.0"
+    "graphql": "^16.5.0"
   }
 }

--- a/query-planner-js/CHANGELOG.md
+++ b/query-planner-js/CHANGELOG.md
@@ -2,7 +2,7 @@
 
 This CHANGELOG pertains only to Apollo Federation packages in the 2.x range. The Federation v0.x equivalent for this package can be found [here](https://github.com/apollographql/federation/blob/version-0.x/query-planner-js/CHANGELOG.md) on the `version-0.x` branch of this repo.
 
-- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions`
+- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions` [PR #2060](https://github.com/apollographql/federation/pull/2060)
 
 ## 2.1.0-alpha.3
 

--- a/query-planner-js/CHANGELOG.md
+++ b/query-planner-js/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 This CHANGELOG pertains only to Apollo Federation packages in the 2.x range. The Federation v0.x equivalent for this package can be found [here](https://github.com/apollographql/federation/blob/version-0.x/query-planner-js/CHANGELOG.md) on the `version-0.x` branch of this repo.
 
+- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions`
+
 ## 2.1.0-alpha.3
 
 - Add `@defer` support [PR #1958](https://github.com/apollographql/federation/pull/1958)

--- a/query-planner-js/package.json
+++ b/query-planner-js/package.json
@@ -32,6 +32,6 @@
     "pretty-format": "^28.0.0"
   },
   "peerDependencies": {
-    "graphql": "^16.0.0"
+    "graphql": "^16.5.0"
   }
 }

--- a/subgraph-js/CHANGELOG.md
+++ b/subgraph-js/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 This CHANGELOG pertains only to Apollo Federation packages in the 2.x range. The Federation v0.x equivalent for this package can be found [here](https://github.com/apollographql/federation/blob/version-0.x/subgraph-js/CHANGELOG.md) on the `version-0.x` branch of this repo.
 
+- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions`
+
 ## 2.1.0-alpha.3
 
 -  Remove dependency on apollo-server-types [PR #2037](https://github.com/apollographql/federation/pull/2037)

--- a/subgraph-js/CHANGELOG.md
+++ b/subgraph-js/CHANGELOG.md
@@ -2,7 +2,7 @@
 
 This CHANGELOG pertains only to Apollo Federation packages in the 2.x range. The Federation v0.x equivalent for this package can be found [here](https://github.com/apollographql/federation/blob/version-0.x/subgraph-js/CHANGELOG.md) on the `version-0.x` branch of this repo.
 
-- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions`
+- Update peer dependency `graphql` to `^16.5.0` to use `GraphQLErrorOptions` [PR #2060](https://github.com/apollographql/federation/pull/2060)
 
 ## 2.1.0-alpha.3
 

--- a/subgraph-js/package.json
+++ b/subgraph-js/package.json
@@ -28,6 +28,6 @@
     "@apollo/federation-internals": "file:../internals-js"
   },
   "peerDependencies": {
-    "graphql": "^16.0.0"
+    "graphql": "^16.5.0"
   }
 }

